amazon-kinesis-client/src/main/java/software/amazon/kinesis/coordinator/DiagnosticEvent.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright 2019 Amazon.com, Inc. or its affiliates.
+ * Licensed under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package software.amazon.kinesis.coordinator;
+
+import software.amazon.kinesis.annotations.KinesisClientInternalApi;
+
+/**
+ * An interface to implement various types of stateful events that can be used for diagnostics throughout the KCL.
+ */
+@KinesisClientInternalApi
+public interface DiagnosticEvent {
```

Comment:
Do not mark interfaces as `@KinesisClientInternalApi`, that is supposed to be for implementations.

Reply:
Removed `@KinesisClientInternalApi` annotation